api/server/router/delta/delta_routes.go

```diff
@@ -0,0 +1,27 @@
+package delta
+
+import (
+	"net/http"
+
+	"github.com/docker/docker/api/server/httputils"
+	"github.com/docker/docker/api/types"
+	"golang.org/x/net/context"
+)
+
+func (d *deltaRouter) postDeltasCreate(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
```

Comment:
It looks good, but I would expect it to be a context dependent. 
Something like that:
https://github.com/moby/moby/blob/b248de7e332b6e67b08a8981f68060e6ae629ccf/distribution/pull_v2.go#L63